plugins/inputs/gnmi/gnmi.go

```diff
@@ -90,6 +91,12 @@ type TagSubscription struct {
 	Elements []string
 }
 
+// Allow to convey additionnal configuration elements to limit the number of arguments (lint)
+type AdditionalConf struct {
+	Trace     bool
+	VendorExt []string
+}
+
```

Comment:
A random struct with just logging and the vendor extensions seems a little random as to its contents. What do you think about these options:
1) ignore the lint error
2) create a configuration struct that has everything except the address to pass around?

Reply:
I'm going through option 2